apps/interactor/docker/harpoon.yml

```
@@ -13,19 +13,20 @@ images:
     commands:
       - FROM python:3.8.1-slim
 
-      - ADD apps/interactor/command /project/
       - ADD apps/interactor /project/interactor
       - ADD modules /project/modules
 
       - WORKDIR /project/config
 
       - - RUN
-        - apt-get update
-          && apt-get install gcc -y
+        - ln -s /project/interactor/command /project/command
+          && apt-get update
+          && apt-get install curl gcc -y
           && pip install pip -U && pip install /project/modules /project/interactor
           && apt-get purge -y gcc
           && apt-get autoremove -y
           && rm -rf /var/lib/apt/lists/*
 
       - ENV INTERACTOR_HOST 0.0.0.0
-      - CMD lifx lan:interactor
+      - HEALTHCHECK CMD lifx --silent --task healthcheck || exit 1
```

I'd rather not use the --task option
```suggestion
      - HEALTHCHECK CMD lifx interactor_healthcheck --silent
```

and the exit 1 seems unnecessary and just means the healthcheck would always fail?

---

The `|| exit 1` is to normalise the exit for Docker's `HEALTHCHECK` directive in case additional exit codes are used by `lifx` in future. I personally caused a few `22`'s, so I know it's possible.

---

And If I specify `--silent` then I have to use `--task` or else it doesn't work. I guess I could just redirect all output to `/dev/null` instead.

---

you have to have the --silent after the task